leadbutt.py

```diff
@@ -218,6 +394,47 @@ def get_metric_statistics(**kwargs):
             output_results(results, this_metric, options)
             time.sleep(kwargs.get('interval', 0) / 1000.0)
 
+        # get enhanced monitoring if it is enabled
+        if enhanced_monitoring:
+            options = get_options(config_options, None, cli_options)
+            # convert minutes to seconds
+            period_local = options['Period'] * 60
+            count_local = options['Count']
+            log_group = enhanced_monitoring['LogGroup']
+            # determine formatter
+            if 'Formatter' in enhanced_monitoring:
```

Comment:
I don't get the difference between `ListFormatter` `Formatter`. Can you add an explanation to the README?